django/applications/catmaid/control/sampler.py

```diff
@@ -963,15 +965,15 @@ def add_multiple_sampler_domains(request, project_id, sampler_id):
             "ends": [{
                 "id": e.id,
                 "node_id": e.end_node_id
-            } for e in domain_ends]
+            } for e in domain_ends] # FIXME: domain_ends is not defined in this function
```

Comment:
This line should be right before the `append()` call (L956): `domain_ends = d.samplerdomainend_set.all()`